src/Pages/DashboardPage/DashboardPage.jsx

```diff
@@ -0,0 +1,73 @@
+import { useAuthContext } from "Context";
+import { useDocTitle } from "Hook/useTitle";
+import { LandingPage } from "Pages";
+import { useState } from "react";
+import { Link } from "react-router-dom";
+import "./DashboardPage.css";
+
+export const DashboardPage = () => {
+  useDocTitle("Dashboard | StormPoint");
+  const encodedToken = localStorage.getItem("StormPlayToken");
+  const [route, setRoute] = useState("profile");
+  const { userState, signout } = useAuthContext();
+
+  return (
+    <section className="showcase-container">
+      <div className="dashboard">
+        <h1 className="dashboard-header">Account</h1>
+        <div className="dashboard-container">
+          <div className="dashboard-sections">
+            <ul>
+              <li
+                onClick={() => setRoute("profile")}
+                className={route === "profile" ? "active" : null}
+              >
+                Profile
+              </li>
+              <li
+                onClick={() => setRoute("settings")}
+                className={route === "settings" ? "active" : null}
+              >
+                Settings
+              </li>
+            </ul>
+          </div>
+          <div className="dashboard-section-details">
+            <div className="profile-details">
+              {route === "profile" ? (
```

Comment:
Instead of using the ternary operator, I think, you can use `&&` here. As you are not doing in the `false` condition other than returning `null`.

Reply:
Thanks for pointing it out !